### PR TITLE
add new JSONLITE serializer to rpc

### PR DESCRIPTION
**Pull request by mvocu**
Adds JsonSerializerJSONLITE serializer, which omits all audit-related properties (*{created|modified}{By|At}*, beanName).

### CODE REVIEW

File: perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+		mixinMap.put(Thanks.class, CabinetMixIn.class);
+		mixinMap.put(Task.class, TaskMixIn.class);
+
+		mapper.setMixIns(mixinMap);
+	}
+
+	private static final JsonFactory jsonFactory = new JsonFactory();
+
+	static {
+		//FIXME odstraneno disable(JsonGenerator.Feature.FLUSH_PASSED_TO_STREAM)
```
Comment (stavamichal) [inline perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java:172]: I know it is a copy, but we should use English here.

File: perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+	}
+
+	@Override
+	public String getContentType() {
+		return CONTENT_TYPE;
+	}
+
+	@Override
+	public void write(Object object) throws IOException {
+		JsonGenerator gen = jsonFactory.createJsonGenerator(out, JsonEncoding.UTF8);
```
Comment (stavamichal) [inline perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java:196]: Method "jsonFactory.createJsonGenerator()" is deprecated. Is there any chance to change that without great effort?

**Review by stavamichal — CHANGES_REQUESTED** [review]

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+/**
+ * JSON lite serializer which strips auditing data from Perun beans.
+ *
+ * @author Pavel Zlámal <zlamal@cesnet.cz>
+ * @author Michal Voců <vocu@cesnet.cz>
+ */
+public final class JsonSerializerJSONLITE implements Serializer {
+
+	@JsonIgnoreProperties({
```
[the remark this answers is not in the document]
Comment (mvocu) [reply perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java:37]: Added.

File: perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java (hunk 1 above)
Comment (mvocu) [reply perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java]: Updated.

File: perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java (hunk 2 above)
Comment (mvocu) [reply perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java]: Done.

**Review by zlamalp — APPROVED** [review]

**Review by stavamichal — APPROVED** [review]

**Pull request merged**